components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/model/AuthProperty.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.action.management.model;
+
+/**
+ * Authentication Property.
+ */
+public class AuthProperty {
+
+    private String name;
+    private String value;
+    private boolean isConfidential;
+
+    public AuthProperty() {
+
+    }
+
+    public AuthProperty(String name, String value, boolean isConfidential) {
```

Comment:
Ideally you should pass the builder to the constructor and build using the builder passed.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5795/commits/d2420ad5643697c5521872bf43d17fe17dd12580